packages/services/hmi-server/src/main/resources/application.properties

```diff
@@ -21,7 +21,7 @@ mp.openapi.extensions.smallrye.info.description=REST endpoints for the TERArium
 ########################################################################################################################
 # Microservice configurations
 ########################################################################################################################
-%dev.data-service/mp-rest/url=http://localhost:3020
+%dev.data-service/mp-rest/url=http://localhost:8001
```

Comment:
Do NOT add this file this *WILL* brake the app.
You can now use the container version in the cluster.